### ISSUE TITLE
v55 build fails on Hera

### ISSUE THREAD

**Issue by zacharyburnettNOAA** [noaa-ocs-modeling/ADC-WW3-NWM-NEMS#19]
In branch `feature/adcirc_v55`, at `/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS`, when running the following command:
```bash
sh build.sh --plat "hera" --components "ADCIRC ATMESH WW3DATA"
```

the build fails with the following error:
```
make[1]: Leaving directory `/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/work'
ifort -O3 -i4   -c -o estofs_tide_fac.o estofs_tide_fac.f
ifort -mkl    -o estofs_tide_fac estofs_tide_fac.o
rm -f estofs_tide_fac.o
echo No module selected. ; cd /scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/thirdparty/nuopc ; exec make COMP_SRCDIR="/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC" COMP_BINDIR="/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC_INSTALL" MACHINE_ID="linux" -f makefile.adc_cap.nuopc nuopcinstall \
          DESTDIR=/ "INSTDIR=/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC_INSTALL"
No module selected.
make[1]: Entering directory `/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/thirdparty/nuopc'
makefile.adc_cap.nuopc:18: Environment variable ADCDIR was not set.
Setting to /scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC

c     -I. -I/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/work/odircp -I/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/prep -I/apps/netcdf/4.6.1/intel/16.1.150/include    -I/apps/hdf5/1.10.4/intel/16.1.150/include    -L. -L/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC  -L/apps/netcdf/4.6.1/intel/16.1.150/lib -L/apps/hdf5/1.10.4/intel/16.1.150/lib -DWITHPETLISTS_on  -g  -traceback  -DESMF_VERSION_MAJOR= couple2swan_modif.F
make[1]: c: Command not found
make[1]: [couple2swan_modif.o] Error 127 (ignored)

c     -I. -I/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/work/odircp -I/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/prep -I/apps/netcdf/4.6.1/intel/16.1.150/include    -I/apps/hdf5/1.10.4/intel/16.1.150/include    -L. -L/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC  -L/apps/netcdf/4.6.1/intel/16.1.150/lib -L/apps/hdf5/1.10.4/intel/16.1.150/lib -DWITHPETLISTS_on  -g  -traceback  -DESMF_VERSION_MAJOR= adc_mod.F90
make[1]: c: Command not found
make[1]: [adc_mod.o] Error 127 (ignored)

c     -I. -I/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/work/odircp -I/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/prep -I/apps/netcdf/4.6.1/intel/16.1.150/include    -I/apps/hdf5/1.10.4/intel/16.1.150/include    -L. -L/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC  -L/apps/netcdf/4.6.1/intel/16.1.150/lib -L/apps/hdf5/1.10.4/intel/16.1.150/lib -DWITHPETLISTS_on  -g  -traceback  -DESMF_VERSION_MAJOR= adc_cap.F90
make[1]: c: Command not found
make[1]: [adc_cap.o] Error 127 (ignored)
ar rv libadc_cap.a adc_cap.o adc_mod.o couple2swan_modif.o
ar: adc_cap.o: No such file or directory
make[1]: *** [libadc_cap.a] Error 1
make[1]: Leaving directory `/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC/thirdparty/nuopc'
make: *** [/scratch2/COASTAL/coastal/save/shared/repositories/ADC-WW3-NWM-NEMS/ADCIRC_INSTALL/adcirc.mk] Error 2
```

The build appears to have completed ADCIRC, but not built NEMS.

**Comment by zacharyburnettNOAA** [comment]
Takis: use compiler option `"intel"` because it defaults to GCC
`sh build.sh --clean -2 --plat "hera" --compiler "intel" --component "ADCIRC ATMESH WW3DATA"`

**Comment by zacharyburnettNOAA** [comment]
new patch file made with @pvelissariou1 :
[ADC-WW3-NWM-NEMS-dev-all_20210305.patch.txt](https://github.com/noaa-ocs-modeling/ADC-WW3-NWM-NEMS/files/6093134/ADC-WW3-NWM-NEMS-dev-all_20210305.patch.txt)

**Comment by zacharyburnettNOAA** [comment]
build errors: [build_adcv55_takis_patch.log](https://github.com/noaa-ocs-modeling/ADC-WW3-NWM-NEMS/files/6093191/build_adcv55_takis_patch.log)

**Comment by zacharyburnettNOAA** [comment]
from the log:
```
[1;31mLmod has detected the following error: [0m The following module(s) are unknown: "esmf"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore-cache load "esmf"

Also make sure that all modulefiles written in TCL start with the string #%Module
```

**Comment by zacharyburnettNOAA** [comment]
fixed with ef6992e0ff868e50b86f0104e8ea1c12be03f43d
